### PR TITLE
[WIP] Install register element wrapper

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
cc @dvoytenko

File: templates/installer.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cat templates/before.installer node_modules/document-register-element/build/document-register-element.max.js templates/after.installer > node_modules/document-register-element/build/document-register-element-installer.max.js
```
Comment (erwinmombay) [inline templates/installer.sh]: might need to switch to node for windows

File: src/amp.js [hunk 2]
```diff
@@ -89,6 +89,12 @@ try {
   throw e;
 }
 
+const iframe = self.document.createElement('iframe');
```
Comment (dvoytenko) [inline src/amp.js]: This is just a test, right?

Comment (erwinmombay) [inline src/amp.js]: yes

File: templates/before.installer [hunk 3]
```diff
@@ -0,0 +1 @@
+self.installDocumentRegisterElement = function(window, document) {
```
Comment (dvoytenko) [inline templates/before.installer]: Is this enough? What about `Object` and `REGISTER_ELEMENT` params?

File: templates/installer.sh (hunk 1 above)
Comment (dvoytenko) [inline templates/installer.sh]: I think, yes.

File: templates/before.installer (hunk 3 above)
Comment (erwinmombay) [inline templates/before.installer]: probably will need to, I'm waiting to match whatever is published and match it

Comment (dvoytenko) [inline templates/before.installer]: SG

**Comment by mrjoro** [comment]
Closing since this hasn't been updated in a while; reopen if needed.

**Pull request closed without merging**